notebooks/tutorials/visualization/blender/landlab_to_blender.ipynb

```diff
@@ -0,0 +1,362 @@
+   "metadata": {},
+   "source": [
+    "## Reading the output file into Blender\n",
+    "\n",
+    "*(Note that this tutorial is written around Blender version 3.6.2 for MacOS; future versions may appear different from the screenshots below)*\n",
+    "\n",
+    "1. Open Blender. By default the session starts with a single object: a cube. Type `x` to delete the cube, and verify this in the pop-up window that asks whether you really want to delete it.\n",
+    "\n",
+    "2. From the `File` menu select `Import` then `Wavefront (.obj)`. Locate a file called `test_mts.obj` in the same folder as this notebook, and select it.\n",
+    "\n",
+    "<div>\n",
+    "    <img src=\"assets/after_initial_import.png\" width=500>\n",
+    "</div>\n",
+    "\n",
+    "3. Before we go any further, it will be helpful to change the *clipping plane* to make our image completely visible. In the upper right of the main view pane, look for a small right-pointing chevron (just to the right of a colorful set of axes labeled X Y Z, see red circle in image below):\n",
+    "\n",
+    "<div>\n",
+    "    <img src=\"assets/view_pane_access.png\" width=500>\n",
+    "</div>\n",
+    "\n",
+    "4. Click this to open a small window, then select the View tab next to it:\n",
+    "\n",
+    "<div>\n",
+    "    <img src=\"assets/clip_extent.png\" width=500>\n",
+    "</div>\n",
+    "\n",
+    "5. Change the value under \"End\" from 1000 m to 10000 m.\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Viewing the model in Blender\n",
+    "\n",
+    "*(Note 1: Blender is a powerful 3D modeling package, and the following only scratches the surface a little bit to help you get started!)*\n",
+    "\n",
+    "*(Note 2: To do the next steps, it really helps to have a mouse with a scroll wheel)*\n",
+    "\n",
+    "* Zoom in and out of the image with the mouse scroll wheel (trying zooming out until the entire terrain is visible in the window)\n",
+    "\n",
+    "* Rotate your view of the image by holding down the middle mouse button / scroll wheel and dragging\n",
+    "\n",
+    "* Pan your view of the image by holding down the shift key and the middle mouse button / wheel and dragging while the image is selected\n",
+    "\n",
+    "### Some default keyboard shortcuts\n",
+    "\n",
+    "#### On the numeric keypad (if your keyboard has one)\n",
+    "\n",
+    "2, 4, 6, 8 = rotate view down / left / right / up (respectively)\n",
+    "\n",
+    "0 = place the viewpoint at the position of the camera\n",
+    "\n",
+    "1, 3, 7 = snap view along y-axis / x-axis / z-axis (respectively) (notice how the guide axes in upper right change)\n",
+    "\n",
+    "5 = toggle between perspective and orthographic view\n",
+    "\n",
+    "9 = rotate view 180 degrees\n",
+    "\n",
+    "#### On the \"main\" part of the keyboard\n",
+    "\n",
+    "Tab = toggle between object mode and edit mode\n",
```

Comment:
Clarify that you first need to hit tab to toggle into edit mode before any of the following shortcuts will do anything.

Reply:
Note added